vm/src/builtins/mappingproxy.rs

```diff
@@ -158,6 +186,7 @@ impl AsMapping for PyMappingProxy {
 impl AsSequence for PyMappingProxy {
     const AS_SEQUENCE: PySequenceMethods = PySequenceMethods {
         contains: Some(|seq, target, vm| Self::sequence_downcast(seq)._contains(target, vm)),
+        length: Some(|seq, vm| Self::sequence_downcast(seq).len(vm)),
```

Comment:
```suggestion
```
In cpython, `mappingproxy` did not offer `length` field for sequence protocol.
I don't know the exact difference between length in mapping protocol and in sequence protocol, so I'm wondering if it is correct whether remove this field or not. @youknowone How do you think? 😊

Reply:
when we don't understand situation very well, following cpython will be the way.